src/spectra/getSpectraShifts.js

```diff
@@ -0,0 +1,33 @@
+import mean from 'ml-array-mean';
+import { gsd } from 'ml-gsd';
+import { xFindClosestIndex } from 'ml-spectra-processing';
+
+/**
+ * Calculates the shift between the signal of interest and a target signal
+ * @param {number} [targetPoint] - Target point to set the interest signal.
+ * @param {number} [from] - Beginning of the range where the interest signal is localed
+ * @param {number} [to] - End of the range where the interest signal is localed
+ * @param {Object} [options={}]
+ * @param {number} [options.minMaxRatio=0.4] - GSD Threshold to determine if a given peak should be considered as a noise.
+ * @return {Array<number>} Array with the shifts between the spectra and the target point
+ */
+
+export function getSpectraShifts(spectra, targetPoint, from, to, gsdOptions) {
```

Comment:
Please use 
calculateSpectraXShifts(spectra, from, to, options = {}) 
Where options is:
- targetX=0 (by default)
- gsd: object that can be directly send to GSD
- nbPeaks=1 (by default): number of peaks to consider